app/assets/stylesheets/common.scss.erb

```diff
@@ -13,7 +13,7 @@
   font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
   font-size: $font-size-base;
   overflow-y: hidden;
-  @include shadow(3);
+  border: 1px solid #5f574F;
```

Comment:
FWIW, I do see that this was used below (although I don't think in a border context).  We should probably fix that too, but don't feel obligated to include that in this PR.